src/test/java/com/synopsys/integration/alert/util/DatabaseConfiguredFieldTest.java

```diff
@@ -8,58 +8,29 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.springframework.beans.factory.annotation.Autowired;
 
 import com.synopsys.integration.alert.common.database.BaseConfigurationAccessor;
 import com.synopsys.integration.alert.common.database.BaseDescriptorAccessor;
 import com.synopsys.integration.alert.common.descriptor.Descriptor;
-import com.synopsys.integration.alert.common.descriptor.DescriptorMap;
 import com.synopsys.integration.alert.common.enumeration.ConfigContextEnum;
 import com.synopsys.integration.alert.common.exception.AlertDatabaseConstraintException;
 import com.synopsys.integration.alert.database.api.configuration.ConfigurationAccessor.ConfigurationModel;
 import com.synopsys.integration.alert.database.api.configuration.ConfigurationFieldModel;
 
 public abstract class DatabaseConfiguredFieldTest extends AlertIntegrationTest {
 
-    private final List<Descriptor> descriptors;
+    private final List<Descriptor> descriptors = new ArrayList<>();
```

Comment:
If the size of the list isn't known or can grow to a large arbitrary size think about using LinkedList to avoid the expense of array copies if the list hits the threshold and need to be resized.  If we know the size of the list or can calculate it then ArrayList is good.  This is just a general comment.  The number of descriptors we currently have is less than the default size of an ArrayList.